main.go

```diff
@@ -5,23 +5,23 @@ import (
 	gp "github.com/Never-M/MyGossip/pkg/gossiper"
 )
 
-func main()  {
+func main() {
```

Comment:
might wanna just convert this to a e2e test file.

./test/e2e/heart_beat_test.go

Reply:
I discussed this with Fuyao. He said he didn't have enough time to do this. So maybe ignore this for now?